src/main/java/com/amuzil/omegasource/magus/skill/conditionals/enums/ActivationType.java

```diff
@@ -0,0 +1,12 @@
+package com.amuzil.omegasource.magus.skill.conditionals.enums;
+
+/**
+ * 4 main types of activation. Each of these have sub-types. Again, I'll define them as an enum for now and
+ * consolidate for good code practice later.
+ */
+public enum ActivationType {
```

Comment:
Why put this and all of the other enums in their own individual class? Most of them are placeholders/reminding us how the system works, but I'd just put them all together in one class.